tests/playbooks/tests_ethtool_coalesce.yml

```
@@ -0,0 +1,91 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    interface: lsrcoalesce1
+    type: veth
+  tasks:
+    - block:
+        - name: >-
+            TEST: I can create a profile without any coalescing option.
+          debug:
+            msg: "##################################################"
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                type: "{{ type }}"
+                autoconnect: no
+                ip:
+                  dhcp4: no
+                  auto6: no
+        - name: Get profile's coalescing options
+          command: "grep ^ETHTOOL_OPTS /etc/sysconfig/network-scripts/ifcfg-{{ interface }}"
+          register: no_coalesce
+          ignore_errors: yes
+        - name: "ASSERT: The profile does not contain coalescing options"
+          assert:
+            that: no_coalesce.stdout == ""
+        - name: >-
+            TEST: I can set rx-frames and tx-frames.
+          debug:
+            msg: "##################################################"
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                type: "{{ type }}"
+                autoconnect: no
+                ip:
+                  dhcp4: no
+                  auto6: no
+                ethtool:
+                  coalesce:
+                    rx_frames: 128
+                    tx_frames: 64
+        - name: Get profile's coalescing options
+          command: "grep ^ETHTOOL_OPTS /etc/sysconfig/network-scripts/ifcfg-{{ interface }}"
+          register: with_coalesce
+        - name: Assert coalesce options set in profile
+          assert:
+            that:
+              - >-
+                  'ETHTOOL_OPTS=\"-C lsrcoalesce1 rx-frames 128 tx-frames 64\"' ==
+                  with_coalesce.stdout
+        - name: "TEST: I can clear coalescing options"
+          debug:
+            msg: "##################################################"
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                type: "{{ type }}"
+                autoconnect: no
+                ip:
+                  dhcp4: no
+                  auto6: no
+        - name: Get profile's coalescing options
+          command: "grep ^ETHTOOL_OPTS /etc/sysconfig/network-scripts/ifcfg-{{ interface }}"
+          register: profile
+          ignore_errors: yes
```

```suggestion
          failed_when: profile.rc not in [0, 1]
```
This should work here, see https://github.com/ansible/ansible/issues/70090#issuecomment-644811266